utilities/src/main/resources/onetable-client-defaults.yaml

```diff
@@ -1,14 +1,30 @@
-# This file contains the default configuration of known OneTable tableFormatsClients for both source and target
-# table formats. The config maps a table format name to a client class, and contains the default configuration
-# values needed by the client.
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+## This file contains the default configuration of known OneTable tableFormatsClients for both source and target
```

Comment:
Is the double `#` intentional here?

Reply:
Yes, the purpose of doing this is to preserve comments and not confuse them with `#` in the license header